src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy.model;
+
+import java.util.Arrays;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
Let’s make the email unique. We need to check the email during user registration and make the email a unique field in the database.